shell.nix

```diff
@@ -5,8 +5,25 @@ in
 pkgs.mkShell {
   name = "scripts-shell";
   buildInputs = with pkgs; [
+    chart-testing
+    ginkgo
+    git
+    go_1_19
+    golint
+    kubectl
+    kubernetes-helm
+    gnumake
+    minikube
     semver-tool
     yq-go
-    chart-testing
   ];
-}
\ No newline at end of file
+  shellHook = ''
+    export HOME=${builtins.getEnv "HOME"}
```

Comment:
why is this needed?

Reply:
In GitHub actions it's required to set HOME specifically, it will fail if you don't set that (in both action file and shell.nix)

Reply:
You can checkout previous commits action runs

Reply:
Interesting, was not aware of this :+1: